uint8 transpoter_id
uint8 transpoter_location
uint8 transpoter_shoes
